app.rb

```diff
@@ -1,9 +1,36 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
+	get '/' do
+		'haha gotcha, try type in "/posts" up there&#9757;&#9757;&#9757;&#9757;'
+	end
   get '/test' do
     'Test page'
   end
 
+  get '/posts' do
+	  @peeps = Peep.all
+  	erb :index
+  end
+
+  get '/posts/new' do 
+  	erb :"posts/new"
+  end
+
+  post '/posts' do 
+  	posted_date = Time.now.strftime("%d/%m/%Y %H:%M")
+  	Peep.create(message: params[:message], date: posted_date)
+  	redirect '/posts'
+  end
+
+  get '/posts/search' do 
+  	erb :"posts/search"
+  end
+  post '/posts/search' do 
+  	@peeps = Peep.search(params[:keywords])
```

Comment:
I can see why you've chosen a post here because the browser is sending a parameter to the server. In this case it would be more conventional to use a get request because this method is reading data from the server and doesn't add any new data to the database. In general POST requests are used when new data is added to the server and GET requests are used when data is read from the server. 

But this will work and looks great 👍